lib/src/internal/plugin.dart

```diff
@@ -665,6 +665,16 @@ class PhotoManagerPlugin with BasePlugin, IosPlugin, AndroidPlugin, OhosPlugin {
 
     return null;
   }
+
+  Future<PermissionState> getPermissionState(
+    PermissionRequestOption requestOption,
+  ) async {
+    final int? result = await _channel.invokeMethod<int>(
+      PMConstants.mGetPermissionState,
+      requestOption.toMap(),
+    );
+    return PermissionState.values[result ?? 0];
```

Comment:
Is there any chance that we can get a nullable result here? If not, use the non-nullable return.